pkg/apis/dynatrace/v1alpha1/dynakube_types.go

```diff
@@ -0,0 +1,127 @@
+package v1alpha1
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/status"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DynaKubeSpec defines the desired state of DynaKube
+// +k8s:openapi-gen=true
+type DynaKubeSpec struct {
+	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+	BaseDynaKubeSpec `json:""`
```

Comment:
No need for an extra type since it isn't being inherited by anyone else.

The OneAgent CRDs may eventually, but let's leave that problem for later.